mirrord-agent/src/main.rs

```diff
@@ -305,13 +315,13 @@ async fn start_agent() -> Result<(), AgentError> {
                                     }
                                 }
                             },
-                            SnifferOutput::TCPClose(close) => {
-                                debug!("SnifferOutput::TCPClose -> close {:#?}", close);
+                            SnifferOutput::TcpClose(close) => {
+                                debug!("SnifferOutput::TcpClose -> close {:#?}", close);
                                 let peer_ids = state.connections_subscriptions.get_topic_subscribers(close.connection_id);
 
                                 for peer_id in peer_ids {
                                     if let Some(peer) = state.peers.get(&peer_id) {
-                                        match peer.channel.send(DaemonMessage::TCPClose(close.clone())).await {
+                                        match peer.channel.send(DaemonMessage::Tcp(DaemonTcp::Close(close.clone()))).await {
```

Comment:
Same here.

`if let Err(fail) = ...` if we don't care about the `Ok` portion.

Reply:
same x3